upstream.wrap

```diff
@@ -0,0 +1,6 @@
+[wrap-file]
+directory = utf8proc-2.6.0
+
+source_url = https://github.com/JuliaStrings/utf8proc/archive/v2.6.0.tar.gz
+source_filename = v2.6.0.tar.gz
+source_hash = b36ce1534b8035e7febd95c031215ed279ee9d31cf9b464e28b4c688133b22c5
```

Comment:
Please add a provide section: https://mesonbuild.com/Wrap-dependency-system-manual.html#provide-section

```
[provide]
libutf8proc = <declared dependency>
```

Reply:
Another useful and great feature from Meson. Thanks.